pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[tool.black]
+line-length = 79
+target-version = ["py36"]
+include = '\.py?$'
+exclude = '''
+/(
+    thirdparty |
+    \.eggs |
+    \.git |
+    \.hg |
+    \.mypy_cache |
+    \.tox |
+    \.venv |
+    _build |
+    buck-out |
+    build |
+    dist
+)/
+'''
```

Comment:
what is a `.toml` file?

Reply:
PEP 518: https://www.python.org/dev/peps/pep-0518/ basically future Python build system things, but it's a crapshoot as to what supports it and what doesn't. Currently just using it for `black` configuration.